src/backend/storage/buffer/localbuf.c

```diff
@@ -398,10 +398,11 @@ LocalBufferAlloc(SMgrRelation smgr, ForkNumber forkNum, BlockNumber blockNum,
  *	  mark a local buffer dirty
  */
 void
-MarkLocalBufferDirty(Buffer buffer)
+MarkLocalBufferDirty(Buffer buffer, bool hint)
```

Comment:
Can we rename this to something easier to understand later? For example: 
`bool is_wal_change`, and then I guess we will have to change the true and false flags, but that way it might be easier to follow.

Reply:
Done